CRM/Core/I18n/Schema.php

```diff
@@ -81,6 +79,7 @@ public static function makeMultilingual($locale) {
    */
   public static function makeSinglelingual($retain) {
     $domain = new CRM_Core_DAO_Domain();
+    $domain->id = CRM_Core_Config::domainID();
```

Comment:
@colemanw If we are in multilingual mode then we probably want to set the locales field for all of them right e.g. we don't want locales to still be populated for domain_id 2 if domain_id 1 is converted to single lingual right?

Reply:
@seamuslee001 I agree with you but I think that's outside the scope of this PR, as it was already not updating all of them.
But I'd be happy to follow up with another PR to try to keep them in sync.